corehq/apps/analytics/tasks.py

```diff
@@ -63,3 +64,19 @@ def track_built_app_on_hubspot(webuser):
     if vid:
         # Only track the property if the contact already exists.
         _track_on_hubspot(webuser, {'built_app': True})
+
+
+@task(queue='background_queue', acks_late=True, ignore_result=True)
+def track_usage(email, event, properties=None):
```

Comment:
Yeah, I thought about that. I decided to name it generically to match up with the corresponding frontend function, [`analytics.wokflow`](https://github.com/dimagi/commcare-hq/blob/master/corehq/apps/style/templates/style/includes/analytics_all.html#L69) (although I remembered the name of this function wrong, so I'll change this to `track_workflow()`. The thinking here was that if we ever decided to switch away from KISSmetrics, we wouldn't have to change every function call.